spring-boot/admin/src/main/java/eu/coatrack/admin/YggAdminApplication.java

```diff
@@ -31,8 +31,10 @@
 import org.thymeleaf.extras.java8time.dialect.Java8TimeDialect;
 import org.thymeleaf.spring4.SpringTemplateEngine;
 import org.thymeleaf.templateresolver.ITemplateResolver;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
 
 @SpringBootApplication
+@EnableSwagger2
```

Comment:
Is this annotation mandatory? Or is is sufficient to put it just into the specific SwaggerConfig class?

Reply:
So, I checked it now.
What I noticed is that the annotation is only mandatory in the applicationFile, where you start all the config from Spring Boot, so in this case YggAdminApplication.java.
Long Story short, I removed the second annotation from the Swaggerconfig class, so from now on it only uses one annotation in the application file